### PR TITLE
[DW-405] Implement UI for Person doctype

### PR DESCRIPTION
**Pull request by DamZiobro**

### CODE REVIEW

File: repository-data/application/src/main/resources/hcm-config/namespaces/website/biography.yaml [hunk 1]
```diff
@@ -47,8 +53,14 @@ definitions:
             wicket.id: ${cluster.id}.left.item
           /prevpositions:
             /cluster.options:
-              ckeditor.config.appended.json: '{toolbar: [{ name: ''summarytoolbar'',  items:
-                [''Link'',''PickInternalLink'',''Source''] }] }'
+              ckeditor.config.appended.json: '{toolbarGroups: [{ name: ''document''}],
+                extraPlugins: ''balloonpanel,a11ychecker'', allowedContent: ''h2 h3
```
Comment (ngta1) [inline repository-data/application/src/main/resources/hcm-config/namespaces/website/biography.yaml]: just wondering if the ckeditor definitions in this file need to include the latest abbr added recently?

File: repository-data/webfiles/src/main/resources/site/freemarker/common/macro/award.ftl [hunk 2]
```diff
@@ -1,11 +1,47 @@
...
+
+        <p>
+          <#if awards?has_content >
+            ${firstname} has won awards for:
+            <ul>
+              <#list awards as personaward>
+                <div>
+                  <#assign awardnameDisplay = personaward.awardname />
+                  <#if personaward.awardlink?has_content>
+                    <li><a href="${personaward.awardlink.link}" onClick="logGoogleAnalyticsEvent('document click','Event','${personaward.awardlink.link}');" onKeyUp="return vjsu.onKeyUp(event)" title="${awardnameDisplay}"><span itemprop="award">${awardnameDisplay}</span></a></li>
```
Comment (ngta1) [inline repository-data/webfiles/src/main/resources/site/freemarker/common/macro/award.ftl]: With the GA tags, the 2nd param is the Category, and the doc type is usually used here i.e. Publication, Event, Data set.  For this, as its a compound related to the Person doc type, could replace "Event" with "Person". Same applies to other GA tags in this PR which are related to Person.

File: repository-data/webfiles/src/main/resources/site/freemarker/common/macro/releaseinfo.ftl [hunk 3]
```diff
@@ -25,7 +25,7 @@
                               This is part of
                               <#list article.apiservice as service>
                                 <@hst.link hippobean=service var="apiservice"/>
-                                <a temprop="url" href="${apiservice}" onClick="logGoogleAnalyticsEvent('document click','Event','${apiservice}');" onKeyUp="return vjsu.onKeyUp(event)" title="${service.title}"><span itemprop="name">${service.title}</span></a><#sep>,
+                                <a itemprop="url" href="${apiservice}" onClick="logGoogleAnalyticsEvent('document click','Event','${apiservice}');" onKeyUp="return vjsu.onKeyUp(event)" title="${service.title}"><span itemprop="name">${service.title}</span></a><#sep>,
```
Comment (ngta1) [inline repository-data/webfiles/src/main/resources/site/freemarker/common/macro/releaseinfo.ftl]: Related to API workstream, so could use "API" as the 2nd logGoogleAnalyticsEvent param?

File: repository-data/application/src/main/resources/hcm-config/namespaces/website/biography.yaml (hunk 1 above)
Comment (DamZiobro) [reply repository-data/application/src/main/resources/hcm-config/namespaces/website/biography.yaml]: yes - why not - I will add 'abbr' here

File: repository-data/webfiles/src/main/resources/site/freemarker/common/macro/award.ftl (hunk 2 above)
Comment (DamZiobro) [reply repository-data/webfiles/src/main/resources/site/freemarker/common/macro/award.ftl]: good point - I will replace

File: repository-data/webfiles/src/main/resources/site/freemarker/common/macro/releaseinfo.ftl (hunk 3 above)
Comment (DamZiobro) [reply repository-data/webfiles/src/main/resources/site/freemarker/common/macro/releaseinfo.ftl]: (y)

File: repository-data/application/src/main/resources/hcm-config/namespaces/website/biography.yaml (hunk 1 above)
Comment (DamZiobro) [reply repository-data/application/src/main/resources/hcm-config/namespaces/website/biography.yaml]: solved by: https://github.com/NHS-digital-website/hippo/commit/4b1e9ee5a5497df9cca0f908a247b4bac8ad4d1a

File: repository-data/webfiles/src/main/resources/site/freemarker/common/macro/award.ftl (hunk 2 above)
Comment (DamZiobro) [reply repository-data/webfiles/src/main/resources/site/freemarker/common/macro/award.ftl]: solved by: https://github.com/NHS-digital-website/hippo/commit/4b1e9ee5a5497df9cca0f908a247b4bac8ad4d1a

File: repository-data/webfiles/src/main/resources/site/freemarker/common/macro/releaseinfo.ftl (hunk 3 above)
Comment (DamZiobro) [reply repository-data/webfiles/src/main/resources/site/freemarker/common/macro/releaseinfo.ftl]: solved by: https://github.com/NHS-digital-website/hippo/commit/4b1e9ee5a5497df9cca0f908a247b4bac8ad4d1a

**Pull request merged**